staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer_test.go

```diff
@@ -1970,6 +1971,19 @@ func TestReconcileLoadBalancer(t *testing.T) {
 			},
 		},
 	}
+	expectedLb1.Probes = &[]network.Probe{
+		{
+			Name: to.StringPtr("aservice1-" + string(service3.Spec.Ports[0].Protocol) +
+				"-" + strconv.Itoa(int(service3.Spec.Ports[0].Port))),
+			ProbePropertiesFormat: &network.ProbePropertiesFormat{
+				Port:              to.Int32Ptr(lbNodesHealthCheckPort),
+				RequestPath:       to.StringPtr("/healthz"),
+				Protocol:          network.ProbeProtocol(network.ProtocolHTTP),
+				IntervalInSeconds: to.Int32Ptr(5),
+				NumberOfProbes:    to.Int32Ptr(2),
+			},
+		},
+	}
 
```

Comment:
Should these new entries use the updated form that was causing the conflicts?

pointer.String
pointer.Int32

Reply:
Yeah, I missed that. But this probably doesn't matter because this is just a unit test and as long as it doesn't fail (which it doesn't) it shouldn't interfere with cluster's behavior.

Reply:
Nevermind, `verify` job ended up failing because of that, so I pushed a fix.